### ISSUE TITLE
Unable to start hb-service after upgrading on RPI from 1.0.30 to 1.0.31

### ISSUE THREAD

**Issue by jostrasser** [homebridge/homebridge#3259]
### Analysis

Upgraded the package on my RPI but service is not starting anymore after updating from 1.0.30 to 1.0.31.

### Expected Behavior

Homebridge is not starting anymore after update. Never saw this issue before on the same update process.

### Steps To Reproduce

running "apt update && apt upgrade"

### Logs

```markdown
Starting Homebridge...
homebridge.service: Failed to determine supplementary groups: No such process
homebridge.service: Failed at step GROUP spawning /bin/run-parts: No such process
Started Homebridge.
homebridge.service: Failed to determine supplementary groups: No such process
homebridge.service: Failed at step GROUP spawning /opt/homebridge/start.sh: No such process
homebridge.service: Main process exited, code=exited, status=216/GROUP
homebridge.service: Failed with result 'exit-code'.
homebridge.service: Scheduled restart job, restart counter is at 12.
Stopped Homebridge.
```

### Configuration

```json
N/A
```

### Environment

- OS: Debian GNU/Linux 11 (bullseye)
- Node: v18.12.1

### Process Supervisor

hb-service

**Comment by jostrasser** [comment]
FYI: ran a manual downgrade to 1.0.30 via apt, after this step my Homebridge is up and running again.

**Comment by peck4919** [comment]
+1 for me. Same setup also. Homebridge installed as a package via apt on an RPi. I'm running Ubuntu Jammy (Ubuntu 22.04.1 LTS (GNU/Linux 5.15.0-1017-raspi aarch64)). Apt updated to 1.0.31 for Homebridge which crashed it. Rolled it back to 1.0.30 and it started working again. For now I marked that update to be held back so it doesn't happen again.

**Comment by controllero** [comment]
> FYI: ran a manual downgrade to 1.0.30 via apt, after this step my Homebridge is up and running again.

Good Morning, new here, but having the problem as well. How do I do this (the rollback)?

**Comment by controllero** [comment]
Thank you, worked like a charm...

**Comment by rionshin** [comment]
Maybe a stupid question, as HB is now on version 1.5.1 what exactly  RPI is updating from 1.0.30 to 1.0.31 ?
I also updated  hour ago and have some small issues but restarting the RPI fix it.

**Comment by DrDeath** [comment]
Same issue here:

```
sudo apt install homebridge
systemctl status homebridge
● homebridge.service - Homebridge
     Loaded: loaded (/lib/systemd/system/homebridge.service; enabled; vendor preset: enabled)
     Active: activating (auto-restart) (Result: exit-code) since Fri 2022-11-11 13:11:44 CET; 522ms ago
    Process: 585474 ExecStartPre=/bin/run-parts /etc/hb-service/homebridge/prestart.d (code=exited, status=216/GROUP)
    Process: 585475 ExecStart=/opt/homebridge/start.sh (code=exited, status=216/GROUP)
   Main PID: 585475 (code=exited, status=216/GROUP)
        CPU: 8ms
```

also a reboot does not help.

After running
`sudo apt install homebridge=1.0.30`
the homebridge.service Starts as expected again.

**Comment by peck4919** [comment]
I created a duplicate report on the apt package repository.... Linked this one.

**Comment by elyscape** [comment]
Sorry about that, y'all. I have submitted a fix in homebridge/homebridge-apt-pkg#5.

**Comment by jostrasser** [comment]
fix is working fine, thanks! :)
